Localization/pl-PL.hjson

```diff
@@ -1,56 +1,57 @@
 Mods: {
 	CompareItemStats: {
 		StatComparers: {
-			// Damage.DisplayName: Damage
-			// CritChance.DisplayName: Crit
-			// Speed.DisplayName: Speed
-			// Knockback.DisplayName: Knockback
-			// FishingPower.DisplayName: Fishing Power
-			// BaitPower.DisplayName: Bait Power
-			// Defense.DisplayName: Defense
-			// PickaxePower.DisplayName: Pickaxe Power
-			// AxePower.DisplayName: Axe Power
-			// HammerPower.DisplayName: Hammer Power
-			// HealLife.DisplayName: Restored Life
-			// HealMana.DisplayName: Restored Mana
-			// UseMana.DisplayName: Mana Cost
-			// Autoswing.DisplayName: Autoswing
-			// CanCatchLavaCritters.DisplayName: Can Catch Lava Critters
-			// CanFishInLava.DisplayName: Can Fish In Lava
-			// TileBoost.DisplayName: Range
-			// WingFlyTime.DisplayName: Fly Time
-			// WingHorizontal.DisplayName: Horizontal Velocity
-			// WingVertical.DisplayName: Vertical Acceleration
-			// Class.DisplayName: Same Class
-			// Regeneration.DisplayName: Life Regeneration
+			Damage.DisplayName: Obrażenia
+			CritChance.DisplayName: Szansa na cios krytyczny
+			Speed.DisplayName: Szybkość użycia
+			Knockback.DisplayName: Odrzut
+			FishingPower.DisplayName: Moc wędki
+			BaitPower.DisplayName: Moc przynęty
+			Defense.DisplayName: Obrona
+			PickaxePower.DisplayName: Moc kilofa
+			AxePower.DisplayName: Moc siekiery
+			HammerPower.DisplayName: Moc młota
+			HealLife.DisplayName: Odzyskiwane życie
+			HealMana.DisplayName: Odzyskiwana mana
+			UseMana.DisplayName: Użycie many
+			Autoswing.DisplayName: Autoużycie
+			CanCatchLavaCritters.DisplayName: Umożliwia łapanie ognistych stworzeń
```

Comment:
Should be `Można w nią złapać stworzenia lawy` as per vanilla translations file (golden bug net)

Reply:
Done